spec/features/reservations_spec.rb

```diff
@@ -176,6 +176,64 @@
     end
   end
 
+  context 'banned equipment processing' do
+    after(:each) do
+      @user.role = 'patron'
```

Comment:
Same here (see [below](https://github.com/YaleSTC/reservations/pull/1260/files#r32799366)). I also think that the correct role is `normal`, not `patron`?